src/reports/components/report-sections/report-section-factory.tsx

```diff
@@ -36,7 +36,7 @@ export type ReportSectionFactory = {
     HeadSection: ReactFCWithDisplayName;
     BodySection: ReactFCWithDisplayName;
     ContentContainer: ReactFCWithDisplayName;
-    HeaderSection: ReactFCWithDisplayName<SectionProps>;
+    HeaderSection: ReactFCWithDisplayName<SectionProps> | ReactFCWithDisplayName;
```

Comment:
Is this change really necessary? As in, do the typings break if that's not the case?

Reply:
I'm just thinking an empty set of props should be a subset of SectionProps so as a result, this should just work, no?

Reply:
actually it's not necessary